mailer.http.class.php

```diff
@@ -136,6 +136,10 @@ protected function get_request_body()
             $body['content']['attachments'] = $attachments;
         }
 
+        if (isset($body['content']['from']['email']) && SparkPost::is_sandbox($body['content']['from']['email'])) {
```

Comment:
do we need the first condition? when we will we have `isset($body['content']['from']['email'])`  false?